dev-packages/e2e-tests/test-applications/supabase-nextjs/lib/initSupabaseAdmin.ts

```diff
@@ -0,0 +1,19 @@
+import { createClient } from '@supabase/supabase-js';
+import * as Sentry from '@sentry/nextjs';
+
+// These are the default development keys for a local Supabase instance
+const NEXT_PUBLIC_SUPABASE_URL = 'http://localhost:54321';
+const SUPABASE_SERVICE_ROLE_KEY =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImV4cCI6MTk4MzgxMjk5Nn0.EGIM96RAZx35lJzdJsyH-qQwv8Hdp7fsn3W0YpN81IU';
```

Comment:
Is it okay to leave this here? Or should we rather add this to the environment?

Same for `NEXT_PUBLIC_SUPABASE_ANON_KEY`

Reply:
These are default local supabase server keys. I think it's ok to leave them here.